public_html/wp-content/mu-plugins/blocks/webpack.config.js

```diff
@@ -1,67 +1,38 @@
 const path = require( 'path' );
-const webpack = require( 'webpack' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 
-const NODE_ENV = process.env.NODE_ENV || 'development';
-const SOURCE_MAPS = process.env.SOURCE_MAPS ? true : false;
```

Comment:
How does `wp-scripts` handle source mapping?

Reply:
It doesn't, so I pushed a change to bring this back. Unfortunately it looks like `cheap-module-eval-source-map` isn't supported for CSS sourcemaps (at least, not using `MiniCssExtractPlugin`), so it only adds JS sourcemaps. But since that's probably always been the case I think it's fine.

Reply:
Yup, that's great, thanks!

Reply:
I think this'll need to bring back the `npm run debug`  script too, right?

Reply:
No, since it defaults to `development`– `npm start` builds the sourcemaps version. Then `npm run build`, which runs `wp-scripts build`, sets the `NODE_ENV` to `production` and no sourcemaps are built in that case.

I didn't notice any time difference between with or without the sourcemap step, so I don't think we need a separate command for it.

Reply:
Perfect, thanks!